libcnb-test/src/container_context.rs

```diff
@@ -154,7 +312,7 @@ impl<'a> ContainerContext<'a> {
     }
 
     /// # Panics
-    pub fn shell_exec(&self, command: impl AsRef<str>) -> ContainerExecResult {
+    pub fn shell_exec(&self, command: impl AsRef<str>) -> LogOutput {
```

Comment:
Could you add rustdocs that clarifies what this method does? In particular, that it runs inside the existing container, rather than starting a new container?

Reply:
I added some basic docs in f209796.